lib/bundling.js

```diff
@@ -60,12 +60,18 @@ function noop() {
  */
 function computeBundleId(obj, discriminatorFields) {
   var ids = [];
+  var hasIds = false;
   for (var i = 0; i < discriminatorFields.length; ++i) {
     var id = _.at(obj, discriminatorFields[i])[0];
     if (id === undefined) {
-      return undefined;
+      ids.push('');
```

Comment:
Why even push an empty string?

This can cause the output to look weird. ie `",,value1,,value2"`